main.tf

```diff
@@ -92,6 +92,19 @@ resource "azurerm_template_deployment" "postgres-paas" {
   }
 }
 
+resource "azurerm_postgresql_database" "postgres-db2" {
```

Comment:
shouldn't we just approve this resource? what if someone wants a third one?

the other better option would be a map / list of databases rather than a `2`

Reply:
This was just for cft demo-int work, can look at making it more generic